### PR TITLE
V1.3.0.rc1

### PR DESCRIPTION
**Pull request by BradleySappington**
sphinx documentation updates
versioning
release notes

### CODE REVIEW

**Review by mperrin — APPROVED** [review]

File: docs/relnotes.rst [hunk 1]
```diff
@@ -33,9 +33,80 @@ Version 1.3.0
...
-X. Improved PSF models for MIRI imager, in particular including an empirical model for the field-dependent shifts of the cruciform artifact seen at short wavelengths.
+1. Improved support for NIRSpec and MIRI IFU PSF calculations, including addition of a ``mode`` attribute for toggling between imaging mode and IFU mode simulations; an option for much faster (but slightly simplified) IFU datacube PSF calculations. Spectral bandpass information added for the IFU bands for both NIRSpec and MIRI. For NIRSpec, IFU mode PSF outputs are rotated by an additional 90 degrees to match the convention used in pipeline-output s3d datacubes made using the IFUAlign orientation. This extra rotation can be optionally disabled if so desired; see the NIRSpec class docstring.
+2. Improved PSF models for MIRI imager, in particular including an empirical model for the field-dependent shifts of the cruciform artifact seen at short wavelengths.
 
+**What's Changed**
```
Comment (marcio-melendez) [inline docs/relnotes.rst:44]: Add interpixel capacitance effects (IPC) for both distortion extension in the simulated PSF and adding some more per-instrument specializations in setup_sim_to_match_fileby @obi-wan76 in https://github.com/spacetelescope/webbpsf/pull/768

Comment (BradleySappington) [reply docs/relnotes.rst:44]: I'll add it!

Comment (BradleySappington) [reply docs/relnotes.rst:44]: done

**Comment by mperrin** [comment]
@BradleySappington I added the handful of edits I wanted to make to the release notes in a commit I pushed to this branch. That should all be good to go now from my perspective, so I think this is ready to merge on Tues morning. Cheers and have a good weekend.

**Pull request merged**